samples/sensor/adt7420/README.txt

```diff
@@ -0,0 +1,5 @@
+Title: adt7420
+
+Description:
+
+A example using the ADT7420 high accuracy digital temperature sensor.
```

Comment:
I expect @dbkinder will ask for more details, like how to build the sample and what is the expected output.

Reply:
Just looking at the other examples - I'll do the whole thing including reference, wiring building and running. Thanks for your review. I'll push the fixes right now.